docs/cse/records-signals-entities-insights/configure-custom-insight.md

```diff
@@ -8,8 +8,9 @@ keywords:
   - cse
 ---
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
-As described in the [Insight Generation Process](insight-generation-process.md) topic, CSE automatically generates an Insight based on an Entity’s Activity Score, which is the cumulative severity of the unique Signals that have fired on an Entity during a period of time. In some cases, you may want CSE to generate an Insight on some basis other than Entity Activity Scores. For example, you might want an Insight generated
+As described in the [Insight Generation Process](/docs/cse/get-started-with-cloud-siem/insight-generation-process/) topic, CSE automatically generates an Insight based on an Entity’s Activity Score, which is the cumulative severity of the unique Signals that have fired on an Entity during a period of time. In some cases, you may want CSE to generate an Insight on some basis other than Entity Activity Scores. For example, you might want an Insight generated
```

Comment:
Can you merge line 14 (`whenever a particular set of Signals are fired in a particular order`) into this paragraph? it's split into two lines.